src/ncp/ncp_base.hpp

```diff
@@ -588,12 +602,48 @@ class NcpBase
     uint8_t mPreferredRouteId;
 #endif
 
+    uint8_t mCurCommandIID;
+
 #if OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
+    uint8_t mCurTransmitIID;
     uint8_t mCurTransmitTID;
     int8_t  mCurScanChannel;
     bool    mSrcMatchEnabled;
 #endif // OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
 
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
```

Comment:
Is it possible to implement this features as a NCP vendor extension? The platform can call APIs defined in the vendor extension so that the extra IID in `otRadioFrame` may also be eliminated. It may require exposing some functionality in ncp_base for the extension to work though.

Reply:
Can we wrap `uint8_t mIid;` from `otRadioFrame` and related code under MULTIPAN define? Having that element in `struct otRadioFrame`  simplifies the implementation without many changes and make use of the existing callback signature(s).

I am open to exploring NCP vendor extension with some guidance from you, but I would prefer to address that in a later PR.

Reply:
Not possible. We cannot use or refer to `OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE `  (or any other config) in any of platform or public header files (including `platform/radio.h` header) - since they may be included by other modules which don't necessarily see the `OPENTHREAD_CONFIG` macro definitions.

Reply:
Good point, thanks Abtin! 

I'd like to keep the ncp extension in later PRs if that's the preferred way to go.

Reply:
If we are looking to eliminate`mIid` from `struct otRadioFrame`, maybe one option to eliminate it is to come up with something like otPlatRadioGetIiD(otRadioFrame *aFrame) API and call it from ncp base.

I want to keep this on separate PR anyways.